pkg/diffs/diffs.go

```diff
@@ -147,6 +147,32 @@ func getJobsByRepoAndName(presubmits config.Presubmits) map[string]map[string]pr
 	return jobsByRepo
 }
 
+// GetChangedClusterJobs returns presubmits and periodics that have had their cluster field changed.
+func GetChangedClusterJobs(prowMasterConfig, prowPRConfig *prowconfig.Config, logger *logrus.Entry) (config.Presubmits, config.Periodics) {
```

Comment:
@stevekuznetsov I realize I'm coming late, but is there a reason why the cluster comparison couldn't be added to the `GetChanged{Presubmits,Periodics}`?